src/components/organisms/ArticlesTable.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import Link from "next/link";
+import { faPencilAlt, faTrash } from "@fortawesome/free-solid-svg-icons";
+import ArticleService from "../../services/ArticleService";
+import UserService from "../../services/UserService";
+
+function ArticleTable() {
+	const userService = new UserService();
+	const articleService = new ArticleService();
+	const [articles, setArticles] = useState(undefined);
+	const [error, setError] = useState(undefined);
+
+	useEffect(() => {
+		(async () => {
+			try {
+				const userData = await userService.getCurrentUser();
+				const data = await articleService.getAllArticlesByUser(userData.id);
+
+				setArticles(data);
+			} catch ({ message }) {
+				setError(message);
+			}
+		})();
+	}, []);
+
+	return (
+		<section>
+			{error && <div className="uk-alert uk-alert-danger">{error}</div>}
+			{!error && articles?.length >= 1 ? (
+				<table className="uk-table uk-table-small uk-table-divider uk-table-hover">
+					<thead>
+						<tr>
+							<th>Title</th>
+							<th>Created</th>
+							<th>Last Updated</th>
+							<th>Published</th>
+							<th />
+						</tr>
+					</thead>
+					<tbody>
+						{articles.map(article => (
+							<tr key={article.id}>
+								<td>{article.title}</td>
+								<td>{article.createdOn}</td>
+								<td>{article.updatedOn}</td>
+								<td>{article.published ? "True" : "False"}</td>
+								<td className="uk-flex uk-flex-between">
```

Comment:
It would be nice to have some CSS on the edit (pencil) option much like what happens with the social icons at the bottom of the page - currently there is no feedback when hovering over the option.

Similarly with the delete option, while this is being built it's worth removing / disabling this option for now so people know that it's not ready yet to be used rather than the appearance of nothing happening